GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -138,10 +141,8 @@ public int getTeamSize()
                     case "blue":
                         blueTeamSize++;
                         break;
-                    case "purple":
-                        purpTeamSize++;
-                        break;
                     default:
```

Comment:
I made it default because it can be `red` instead of `purple`, and somewhere in the project assigning team is like:
```cs
if (team.ToLower() == "blue)
{ }
else
{ }
```
That's why I made it `default`.